src/electron/windows/electron_builder_signing_plugin.cjs

```diff
@@ -22,7 +22,7 @@
  */
 async function electronBuilderEntryPoint(configuration) {
   // CommonJS module is required, ES6 module is not supported by electron-builder
```

Comment:
this is actually not true, I was able to use electron builder in a ES6 module here: https://github.com/Jigsaw-Code/outline-client/blob/1f3325bfd093aff9356014abd20eefc34b08b8d3/src/electron/build.action.mjs#L18-L19

Reply:
we should turn this into a proper action call in a follow-up PR (e.g. `runAction('electron/windows/sign')` or something)

Reply:
Good catch about the action, let me try it.

And here is the error message from `electron-builder` if I switch to ES6 `mjs`, so we have to do the CommonJS one:

```
/Users/yijunyi/Source/outline-client/node_modules/app-builder-lib/out/platformPackager.js:597
      const m = require(p);
                ^
  Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/yijunyi/Source/outline-client/src/electron/windows/electron_builder_signing_plugin.mjs not supported.
  Instead change the require of /Users/yijunyi/Source/outline-client/src/electron/windows/electron_builder_signing_plugin.mjs to a dynamic import() which is available in all CommonJS modules.
```

Reply:
then why does build.action.mjs work?

if you cant get it to work i would just leave it, my intent is for actions to all be ESM

Reply:
Right, the error is triggered in the internal implementation of electron-builder `node_modules/app-builder-lib/out/platformPackager.js`, it's out of our control. Our own code doesn't have any requirements about CommonJS or ES6.

Reply:
Cool, NVM then